tests/ctst/features/dmf.feature

```diff
@@ -73,3 +73,21 @@ Feature: DMF
     |           Non versioned |           2 |        100 |           1 |
     |               Versioned |           2 |        100 |           1 |
     |               Suspended |           2 |        100 |           1 |
+
+    @2.7.0
+    @PreMerge
+    @Dmf
+    @ColdStorage
+    Scenario Outline: Override of a cold object
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-cold" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-cold"
+    And dmf volume should contain <objectCount> objects
+    Given <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-cold"
+    Then dmf volume should contain 1 objects
```

Comment:
We might want to add a small wait just before this as the transition is not instantaneous. In case we had a bug, we might run the check before the transition completes.

Reply:
We have a sleep here : https://github.com/scality/Zenko/blob/a128297c0421a46a7ffa8ddbc80d6550ff8af9b0/tests/ctst/common/common.ts#L302